ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -27,26 +27,26 @@ CONFIG="${SHARED_DIR}/install-config.yaml"
 STATIC_IPS="${SHARED_DIR}"/static-ip-hosts.txt
 base_domain=$(<"${SHARED_DIR}"/basedomain.txt)
 machine_cidr=$(<"${SHARED_DIR}"/machinecidr.txt)
+ENABLE_TEMPLATE="${SHARED_DIR}"/enable_template_content.txt
 
 MACHINE_POOL_OVERRIDES=""
 RESOURCE_POOL_DEF=""
 set +o errexit
-VERSION=$(echo "${JOB_NAME}" | grep -o -E '4\.[0-9]+')
+release="${installer_bin}" coreos print-stream-json |jq -r '.architectures.x86_64.artifacts.vmware.release'
+VERSION="${release:1:2}"
```

Comment:
I think you can get version info just like https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/check/control-plane-machinesets/cucushift-installer-check-control-plane-machinesets-commands.sh#L8-L22